action.yml

```
@@ -11,6 +11,12 @@ inputs:
   from-title:
     description: 'To parse the issue numbers from title, a boolean'
     required: false
+  labels-to-copy:
```

```suggestion
  include-labels:
```
It's important to get this API right. But however I try, I fail to come up with something that will also be descriptive, simple and categorize the parameter to `labels`.

If I didn't have the context, it'd be sufficient for me to just have `include` as the name for this input.

What do you think? Can you come up with an alternative or a better argument for `labels-to-copy`?

---

`labels-to-include` ?

---

That should be fine! Thanks, I will release this with the other PR together!